Content.Shared/Utility/ContentHelpers.cs

```diff
@@ -102,7 +108,51 @@ public static int RoundToNearestLevels(double actual, double max, int levels)
                     nearest = i;
                 }
             }
+
             return nearest;
         }
+
+        /// <summary>
+        /// Returns the index of an array of <paramref name="size"/> which roughly corresponds to where <paramref name="actual"/> lies
+        /// on on a decimal scale from 0 to <paramref name="max"/>.
+        /// </summary>
+        /// <example>
+        /// Imagine you have an array of 3 images [icon-0, icon-1, icon-2] and stack of 100 wires.
+        /// You want 0-33 wires to correspond to icon-0. And 34-67 wires to correspond to icon-1, etc.
+        /// In this case you would use <code>RoundToNearestIndex(actual, 100, 3)</code>
+        /// </example>
+        /// <param name="actual">The point to be translated into index. If greater than max, it defaults to size-1.
+        /// If lower or equal to 0, it defaults to 0.</param>
+        /// <param name="max">The maximum value of the scale</param>
+        /// <param name="size">The size of array you want to map to.</param>
+        /// <returns>An integer from 0 to <paramref name="size" />-1.</returns>
+        /// <exception cref="ArgumentException">
+        ///     Thrown if size is less than 1.
+        ///     Thrown if max is 0 or less.
+        /// </exception>
+        public static int RoundToNearestIndex(double actual, double max, int size)
```

Comment:
How's this different to the other ones? Seems like the same use case (one will always round 0.1 to 1 and the other will round to 0)

Reply:
I'll double check, but I do remember the RoundToLevels looking a bit off.

Reply:
replaced with `RoundToNearestLevel` but made it faster.